python/paddle/fluid/tests/unittests/test_cumprod_op.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import numpy as np
+
+from op_test import OpTest
+import random
+import paddle
+
+import paddle.nn as nn
+import paddle.nn.functional as F
+import paddle.fluid as fluid
+import paddle.fluid.core as core
+from paddle.fluid import compiler, Program, program_guard
+
+np.random.seed(0)
+
+
+# test function.
+class TestCumprod(OpTest):
+    def setUp(self):
+        paddle.enable_static()
+        self.op_type = "cumprod"
+        self.init_dtype()
+
+        x = (np.random.rand(2, 3, 10, 10) + 0.5).astype(self.dtype)
+        out = np.cumprod(x, axis=1)
```

Comment:
Too few test cases. Your test cases do not cover all cases in the C++ codes. Please add more test cases.